.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] It would be nice if you added `node_modules` too to the `.gitignore` because it can be added by accident while setting up linters